core/trino-main/src/main/java/io/trino/operator/aggregation/TypedSet.java

```diff
@@ -33,6 +33,12 @@
 import static java.lang.String.format;
 import static java.util.Objects.requireNonNull;
 
+/**
+ * A set of unique SQL values stored in a {@link Block}.
+ *
+ * <p>Depending on the factory method used, the values' equality may be
+ * determined using SQL equality or {@code IS DISTINCT FROM} semantics.
```

Comment:
IS NOT DISTINCT FROM ?

Reply:
I phrase it this way because I think it's a bit more consistent than sometimes saying `IS NOT DISTINCT FROM` and sometimes saying `IS DISTINCT FROM`, and I don't think it makes it any less clear, but I could change it.

(And at the nit-picking level, I think phrasing it as "`IS DISTINCT FROM` semantics" covers it; we are using the operator's semantics, though we are reversing its result.)